cmd/erasure.go

```diff
@@ -133,42 +129,21 @@ func getDisksInfo(disks []StorageAPI, local bool) (disksInfo []DiskInfo, errs []
 		if disks[i] == OfflineDisk {
```

Comment:
I think we need to remove this if.. so at least offlineDisks[ep] will be increased in this case.